### PR TITLE
Decouple code coverage from npm test and add npm run cc command

### PR DESCRIPTION
**Pull request by nigma456**
##### Objective
We want to run the code coverage separately from the npm test.

##### Abstractions
Use npm run cc command to do the codecoverage for the automation test
Add argument --codecoverage in npm test script to run code coverage.

"cc": "npm run compile && node ./out/test/runTest.js --codecoverage"

Then run test or code coverage according to this argument:
		if (process.argv[2] === '--codecoverage') {
		 	extensionTestsPath = path.resolve(__dirname, './suite/codeCoverage');
		} else {
			extensionTestsPath = path.resolve(__dirname, './suite/index');
		}

##### Tests performed
Passed local testing and Github CI

### CODE REVIEW

**Review by LanDong-adsk — APPROVED** [review]

File: extension/src/test/runCodeCoverage.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
+
+import { runTests } from 'vscode-test';
+
+async function main() {
```
Comment (LanDong-adsk) [inline extension/src/test/runCodeCoverage.ts:5]: this file is redundant. We can use some input argument to distinguish between regular run and code coverage run.

Comment (nigma456) [reply extension/src/test/runCodeCoverage.ts:5]: I am trying to add an argument to the npm script

Comment (LanDong-adsk) [reply extension/src/test/runCodeCoverage.ts:5]: this file should be deleted.

Comment (LanDong-adsk) [reply extension/src/test/runCodeCoverage.ts:5]: this file should be deleted.

Comment (nigma456) [reply extension/src/test/runCodeCoverage.ts]: Yes, I have added the argument --codecoverage to npm script and removed this file.

**Review by LanDong-adsk — APPROVED** [review]

**Review by Sen-real — COMMENTED** [review]
cool

**Pull request merged**